bookshelf.js

```diff
@@ -0,0 +1,51 @@
+var button = document.createElement('button');
+button.setAttribute('id','fetch-books-btn');
+var textNode = document.createTextNode('Fetch books');
+button.appendChild(textNode);
+button.addEventListener('click', fetchBooks);
+document.body.appendChild(button);
+
+// var fetchBooks = function() 
+function fetchBooks() {
+const booksJSON = 'https://raw.githubusercontent.com/codeyourfuture/bookshelf-project/master/books.json'
+fetch(booksJSON)
+    .then(response => response.json())
+    .then(processBooks)
+     .then(removeBtn)
+    .then(books => {
```

Comment:
I think this last `.then` is unnecessary?

Reply:
true